### DISCUSSION TITLE
Having issues shield read "X open issues"

### DISCUSSION THREAD

**Discussion by andrewtavis** [badges/shields#8661] [Q&A] [answered]
Hello :)

I was wondering if it would be possible to reorder the words of the issues shield message for GitHub such that the whole shield reads `X open issues`? 

I'm currently trying to do monochrome shields for repo readmes ([see example](https://github.com/scribe-org/Scribe-iOS)). I know that if I want to get rid of the label I can add `label=%20`, but then for issue, PR and similar shields we just have `X open` and the like, which isn't very descriptive. I've looked a bit and it doesn't seem to be possible to append text to the end of generated messages, but maybe I just didn't see it 🤔 

If it's not possible atm, would others find this appealing?

**Comment by calebcartwright** [comment, marked as answer]
No I don't believe so. Some semi related discussion in #8708
